src/database/migrations/1745439840451-create-college-table.ts

```diff
@@ -0,0 +1,36 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CreateCollegeTable1745439840451 implements MigrationInterface {
+    name = 'CreateCollegeTable1745439840451'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "college" DROP COLUMN "provost"`);
+        await queryRunner.query(`ALTER TABLE "college" DROP COLUMN "year"`);
+        await queryRunner.query(`ALTER TABLE "college" ADD "yearEstablished" date NOT NULL`);
+        await queryRunner.query(`ALTER TABLE "college" ADD "description" text`);
+        await queryRunner.query(`ALTER TABLE "college" ADD "created_at" TIMESTAMP NOT NULL DEFAULT now()`);
+        await queryRunner.query(`ALTER TABLE "college" ADD "updated_at" TIMESTAMP NOT NULL DEFAULT now()`);
+        await queryRunner.query(`ALTER TABLE "college" ADD "deleted_at" TIMESTAMP`);
+        await queryRunner.query(`ALTER TABLE "college" DROP CONSTRAINT "PK_ebef1972362002203cdf7a22e0c"`);
+        await queryRunner.query(`ALTER TABLE "college" DROP COLUMN "id"`);
+        await queryRunner.query(`ALTER TABLE "college" ADD "id" uuid NOT NULL DEFAULT uuid_generate_v4()`);
+        await queryRunner.query(`ALTER TABLE "college" ADD CONSTRAINT "PK_ebef1972362002203cdf7a22e0c" PRIMARY KEY ("id")`);
+        await queryRunner.query(`ALTER TABLE "college" ALTER COLUMN "uniform" DROP NOT NULL`);
```

Comment:
why are you dropping not null. are you saying each college does not have its own color code